src/main/java/core/basesyntax/service/implementation/CreateReportServiceImplementation.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.service.CreateReportService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CreateReportServiceImplementation implements CreateReportService {
+    private static final int START_INDEX = 3;
+    private OperationStrategy operationStrategy;
+
+    public CreateReportServiceImplementation(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public String createReport(String dataFromFile) {
+        Map<String, Integer> fruitsMap = new HashMap<>();
+        String newLineString;
+
+        newLineString = Arrays.toString(dataFromFile.split(System.lineSeparator()));

```

Comment:
why we declaring new variable without value and in next row assigning it?
```suggestion
        String newLineString = Arrays.toString(dataFromFile.split(System.lineSeparator()));
```